### PR TITLE
Can add custom icons to panel button and even null if you don't want any icon

### PR DESCRIPTION
**Pull request by VeraZab**
screen shot 2018-10-30 at 4 35 59 pm

### CODE REVIEW

**Comment by VeraZab** [comment]
cc @dmt0 @nicolaskruchten is this ok with you guys?

File: src/components/containers/PanelHeader.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ class PanelHeader extends Component {
     const {children, addAction, allowCollapse, toggleFolds, hasOpen} = this.props;
 
     // dropdown is styled with same styles as react-select component - see _dropdown.scss
-    const icon = <PlusIcon />;
+    const icon = 'customIcon' in addAction ? addAction.customIcon : <PlusIcon />;
```
Comment (nicolaskruchten) [inline src/components/containers/PanelHeader.js]: could this be `addAction.customIcon || <PlusIcon />` ?

Comment (nicolaskruchten) [reply src/components/containers/PanelHeader.js]: ah well no, I see.

**Comment by nicolaskruchten** [comment]
In principle yes :)

**Pull request closed without merging**